.travis.yml

```diff
@@ -2,6 +2,11 @@ env:
   global:
     - LC_CTYPE=en_US.UTF-8
 
+branches:
```

Comment:
from my understanding of the docs, that means any pushes to those branches, and any pull requests onto those branches